src/startup.py

```diff
@@ -0,0 +1,48 @@
+from leader import *
+from follower import *
+"""
+Determine if an AR tag is found, then run follower or leader functions.
+"""    
+def find_tag():
+    tag = TagRecognition()
+    data = tag.detect()
+    detected = False
+
+    if data:
+        detected = true
+
+    return detected
+
+
+def follower():
+    follower = Follower()
+    while True:
+        follower.follow()
+
+
+def leader():
+    leader = Leader()
```

Comment:
Variable name also contradicts with the function it's in which contradicts with the module it's using. Your code has leader set up as three different entities and it makes the code hard to follow or differentiate between what is what.